app/mailboxes/support_mailbox.rb

```diff
@@ -21,13 +21,26 @@ def process
   private
 
   def find_channel
-    mail.to.each do |email|
+    find_channel_with_to_mail if @channel.blank?
+    find_channel_with_cc_mail if @channel.blank?
+
+    raise 'Email channel/inbox not found' if @channel.nil?
+
+    @channel
+  end
+
+  def find_channel_with_cc_mail
+    mail.cc.each do |email|
       @channel = Channel::Email.find_by('lower(email) = ? OR lower(forward_to_email) = ?', email.downcase, email.downcase)
       break if @channel.present?
     end
-    raise 'Email channel/inbox not found' if @channel.nil?
+  end
 
-    @channel
+  def find_channel_with_to_mail
+    mail.to.each do |email|
+      @channel = Channel::Email.find_by('lower(email) = ? OR lower(forward_to_email) = ?', email.downcase, email.downcase)
+      break if @channel.present?
+    end
   end
```

Comment:
similar comment as above.  We could move the logic into an `EmailChannelFinder` in `app/finders` i guess. 
so that the chances of people missing out `'lower(email) = ? OR lower(forward_to_email) = ?'` is reduced.